### ISSUE TITLE
Resnet50, batch size and learning rate

### ISSUE THREAD

**Issue by howardchina** [ArdhenduBehera/cap#15]
https://github.com/ArdhenduBehera/cap/blob/cae53050af583c2b3fa38081c0abc227199e7dc1/train_CAP.py#L110
the setting is lr=1e-4 with bs=8.

I reimplement CAP on mmclassification, a open source pytorch based platform.
I tried a larger batch size and 4GPUs.
Every time I try a bigger learning rate with bigger batch size, such as lr=0.008 for bs=64 or lr=0.032 for bs=4GPUs*64, the model failed to converge.

But lr=0.0001 is good to use. Interesting.
